test/spmd/test_xla_virtual_device.py

```diff
@@ -48,7 +48,8 @@ def test_model_weight_metrics(self):
     model = nn.Linear(128, 64).to(xm.xla_device())
     xs.mark_sharding(model.weight, self._get_mesh((1, self.n_devices)),
                      partition_spec)
-    self.assertNotIn("VirtualDeviceUsage", met.counter_names())
+    self.assertIn("VirtualDeviceUsage", met.counter_names())
```

Comment:
@steventk-g This change happens to restore the test results of virtual device back to pre-func period.

Reply:
Thanks for the heads up!

Reply:
Thanks for the fix @alanwaketan